.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  run-tests-for-nodejs:
+    runs-on: ubuntu-latest
```

Comment:
I would suggest using a know version of OS say **ubuntu-20.04** to maintain build stability. Since **ubuntu-latest** will be updated whenever there is a new version available. You might be surprised by build result.